package.json

```diff
@@ -104,7 +104,7 @@
     "@typescript-eslint/parser": "^2.26.0",
     "auto-changelog": "^2.0.0",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.1.0",
+    "babel-loader": "8.1.0",
```

Comment:
Dumb fix for this issue that popped up when running `yarn test` after installing some new dependencies:
```There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "babel-loader": "8.1.0"

Don't try to install it manually: your package manager does it automatically.
However, a different version of babel-loader was detected higher up in the tree:

  /Users/chrisgarcia/Code/Cision/rover-ui/node_modules/babel-loader (version: 8.2.2)

Manually installing incompatible versions is known to cause hard-to-debug issues.
```